app/views/local_authorities/_filter.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="app-view-filter govuk-!-margin-right-5 govuk-!-padding-5">
+  <%= form_with url: local_authorities_path, method: :get do |form| %>
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Filter by",
+      margin_bottom: 4,
+    } %>
+
```

Comment:
Maybe, like with search pages https://www.gov.uk/aaib-reports, we can include a visually hidden heading here and save some more space
`<h2 class="govuk-visually-hidden">Filter</h2>`

Reply:
Header is smaller now, part of the options select.